CI/steam/scripts_macos/launch.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/zsh
+ 
+arch_name="$(uname -m)"
+
+# legacy app installation
+if [ -d OBS.app ]; then
+    exec open OBS.app -W --args "$@"
+fi
+
+if [ "${arch_name}" = "x86_64" ]; then
+    exec open x86/OBS.app -W --args "$@"
```

Comment:
`open` usually exits when `dyld_loader` has finished starting the application - this would usually mean that the shell script exits immediately. Dunno if Steam then still considers the app to be "running".

Reply:
That's what `-W` is for ("Wait until the applications exit").

Steam doesn't track process trees on Mac it would otherwise consider the app to be closed unfortunately.